resin/resin/dialog/file_dialog.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef RESIN_FILE_DIALOG_HPP
+#define RESIN_FILE_DIALOG_HPP
+
+#include <future>
+#include <libresin/utils/logger.hpp>
+#include <nfd/nfd.hpp>
```

Comment:
Is NFD used anywhere in this header file? If not, move this include to cpp file

Reply:
It's a leftover from the previous commits, good catch